rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -296,6 +314,10 @@ static NEVER_INLINE s32 savedata_op(ppu_thread& ppu, u32 operation, u32 version,
 			}
 			case CELL_SAVEDATA_FOCUSPOS_NEWDATA:
 			{
+				//focus new data, placed at the end of the vector.
+				// not currently supported in the ui
```

Comment:
This was emulate adding a new data item to the list. Instead we use a new save data button.

See - https://youtu.be/_f0kC7lgdtM?t=46s